docs/ledger-integration.md

```diff
@@ -135,16 +135,16 @@ Agent Land is our stable, public testnet for the Fetch Ledger v2. As such, most
 
 | Parameter      | Value                                                                      |
 | -------------- | -------------------------------------------------------------------------- |
-| Chain ID       | agent-land                                                                 |
+| Chain ID       | stargateworld-2                                                                 |
```

Comment:
It seems you just did a global agent-land --> stargateworld search&replace. Some of these links do not work.

Also the test above this table needs to be updated as well.